src/main/webapp/WEB-INF/web.xml

```diff
@@ -49,4 +49,14 @@
     <listener-class>org.olf.folio.order.listeners.ConfigurationListener</listener-class>
   </listener>
 
+  <session-config>
+    <session-timeout>525600</session-timeout>
+    <cookie-config>
+        <name>folio-order-import-session</name>
+        <path>/</path>
```

Comment:
I'm not sure if this session-config block is necessary for anyone else.  But this logic depends on saving the username in the session, and by default, our session path was something more complex than '/' and wrong, so I needed to override it here.  I don't think this block causes any harm to include here since it's the first use of sessions within the app (I think), but I may be missing something.